packages/blade/src/components/Carousel/NavigationButton/StyledNavigationButton.native.tsx

```diff
@@ -28,6 +31,7 @@ const StyledNavigationButton = ({
       onPress={onClick}
       onPressIn={() => setIsPressed(true)}
       onPressOut={() => setIsPressed(false)}
+      style={castNativeType(theme.elevation.lowRaised)}
```

Comment:
we added shadow to nav buttons now? it wasn't there before. is this intended?

Reply:
I think it was missed before.